### ISSUE TITLE
Activity Stream: Show entire activity stream

### ISSUE THREAD

**Issue by robredpath** [IATI/ckanext-iati#413]
As a member of IATI Support,
I want to be able to see the entire Activity Stream for a publisher since they first registered
so that I can understand where errors might have been introduced.

I note that this is available by incrementing the page number at the end of the URL (e.g. https://iatiregistry.org/publisher/activity/fcdo/100 ) but no UI element is exposed to allow paging 

**Acceptance criteria**

An admin user (or any user, it doesn't need to be restricted) can access the whole Activity stream using on-page UI elements. 

Optionally, a longer page length would be welcome

2. **Non-functional Criteria** _(Include availability, maintainability, performance, reliability, scalability, security, and usability criteria)_

**Comment by cormachallinanderilinx** [comment]
The pagination used for activity streams is different from the normal pagination in ckan.
To complete this work it would take a quite a bit of hacking and patching.
Pagination for activity streams is available in ckan2.10 and after discussing this internally, we think the best approach might be to focus the effort on going to 2.10

Let me know your thoughts on this

**Comment by cormachallinanderilinx** [comment]
https://github.com/ckan/ckan/issues/6108

**Comment by robredpath** [comment]
> we think the best approach might be to focus the effort on going to 2.10

Agreed. I'm updating #406 to work through the details of upgrading to 2.10
